mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -139,20 +148,33 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 		ARG 3 remainingUseTicks
 	METHOD method_7932 getRarity ()Lnet/minecraft/class_1814;
 	METHOD method_7933 increment (I)V
+		COMMENT Increments the count of items in this item stack.
 		ARG 1 amount
+			COMMENT the amount to increment
 	METHOD method_7934 decrement (I)V
+		COMMENT Decrements the count of items in this item stack.
 		ARG 1 amount
+			COMMENT the amount to decrement
 	METHOD method_7935 getMaxUseTime ()I
 	METHOD method_7936 getMaxDamage ()I
 	METHOD method_7937 parseBlockTag (Ljava/lang/String;)Ljava/util/Collection;
 		ARG 0 tag
 	METHOD method_7938 hasCustomName ()Z
 	METHOD method_7939 setCount (I)V
+		COMMENT Sets the count of items in this item stack.
 		ARG 1 count
+			COMMENT the count of items
 	METHOD method_7940 canDestroy (Lnet/minecraft/class_5415;Lnet/minecraft/class_2694;)Z
 		ARG 1 tagManager
 		ARG 2 pos
-	METHOD method_7941 getSubTag (Ljava/lang/String;)Lnet/minecraft/class_2487;
+	METHOD method_7941 getNbt (Ljava/lang/String;)Lnet/minecraft/class_2487;
```

Comment:
It seems weird to repeat these for `Tag` and `SubTag`. Why not `SubNbt` or similar?
This happens a bunch throughout the class?

Reply:
When I worked on that PR, I was told that the `Sub` part isn't really necessary given the key parameter.
Javadocs are present to reduce the confusion.

Reply:
and now you have 3 types of names for methods working with sub elements